src/Documentation/Documentation.php

```diff
@@ -0,0 +1,97 @@
+<?php
+
+namespace Jobilla\DtoCore\Documentation;
+
+use Illuminate\Support\Collection;
+
+class Documentation
+{
+    /**
+     * @param string $apiPathPrefix
+     * @param string $apiVersion
+     *
+     * @return array
+     */
+    public function get(string $apiPathPrefix, string $apiVersion): array
+    {
+        $routes      = RouteFactory::fromLaravelRoutes($apiPathPrefix);
+        $tags        = $this->getTags($routes);
+        $definitions = $this->getDefinitions($routes);
```

Comment:
Just a suggestion, as you use the variable only once in your code and the method-name is saying exactly what to expect from it, I would call it from within the returning array. Same for the `$path`. `$routes` and `$tags` are used multiple times in your method, so good to have a variable for that.